zeus/device/cpu/rapl.py

```diff
@@ -147,6 +147,12 @@ def __init__(self, message: str) -> None:
         """Initialize Zeus Exception."""
         super().__init__(message)
 
+class ZeusRAPLPermissionError(ZeusBaseCPUError):
+    """Zeus GPU exception that wraps No Permission to perform GPU operation."""
+
+    def __init__(self, message: str) -> None:
+        """Intialize the exception object."""
+        super().__init__(message)
```

Comment:
Would it be reasonable to remove the `_exception_map` and just catch exceptions indivudally since we only really have to deal with `FileNotFoundError` and `PermissionError`?

Reply:
`_exception_map` isn't really part of the abstract class contract anyway, so removing it is fine. It was just there for GPU classes to ease mapping NVML exceptions to Zeus exceptions.